pkg/util/azureclient/keyvault/vaults.go

```diff
@@ -0,0 +1,32 @@
+package keyvault
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	mgmtkeyvault "github.com/Azure/azure-sdk-for-go/services/keyvault/mgmt/2019-09-01/keyvault"
```

Comment:
`pkg/util/azureclient/keyvault` is for wrapping `github.com/Azure/azure-sdk-for-go/services/keyvault/v7.0/keyvault` client.

I guess we should follow the existing pattern for `github.com/Azure/azure-sdk-for-go/services/SERVICE_NAME/mgmt/...` clients and put its wrapper into `github.com/Azure/ARO-RP/pkg/util/azureclient/mgmt/keyvault`?